scalable-integrator/integrator-deployment.yaml

```diff
@@ -82,4 +82,4 @@ spec:
           name: integrator-conf-datasources
       - name: shared-pd
         persistentVolumeClaim:
-          claimName: shared-volume-claim
+          claimName: integrator-server-share-volume-claim
```

Comment:
Shall we name this as ""scalable-integrator-shared-volume-claim" ?

Reply:
+1